_security-analytics/threat-intelligence/api/monitor.md

```diff
@@ -0,0 +1,324 @@
+---
+layout: default
+title: Monitor API
+parent: Threat intelligence APIs
+grand_parent: Threat intelligence
+nav_order: 35
+---
+
+# Monitor API
+
+The threat intelligence Monitor API creates, searches, and updates monitors using the monitor's ID.
+
+
+---
+## Create or update a threat intelligence monitor
+
+Creates or updates a new threat intelligence monitor.
+
+### Path and HTTP methods
+
+The `POST` method creates a new monitor. The `PUT` method updates the monitor.
+
+```json
+POST _plugins/_security_analytics/threat_intel/monitors
+PUT _plugins/_security_analytics/threat_intel/monitors/<monitor_id>
+```
+
+### Request fields
```

Comment:
The table is formatted incorrectly: 
<img width="836" alt="image" src="https://github.com/user-attachments/assets/7b027cd7-9a37-45c1-b12d-80ea9ae4a458">

Reply:
I think I see the problem.